crates/typst/src/foundations/int.rs

```diff
@@ -214,6 +212,78 @@ impl i64 {
             self >> shift
         }
     }
+
+    /// Converts bytes to an integer. The bytes will be treated as a 64-bit signed integer.
+    /// The bytes should be at most 8 bytes (64 bits) in size to fit in a 64-bit integer, otherwise an error will occur.
+    ///
+    /// ```example
+    /// #int.from-bytes(bytes((0, 0, 0, 0, 0, 0, 0, 1)), "little")
+    /// ```
+    #[func]
+    pub fn from_bytes(
+        /// The bytes that should be converted to an integer.
+        bytes: Bytes,
+        /// Endianness of the conversion.
+        endianness: Endianness,
+    ) -> StrResult<i64> {
+        let len = bytes.len();
+        if len > 8 {
+            return Err("Bytes too large to convert to a 64 bit number".into());
```

Comment:
```suggestion
            bail!("too many bytes to convert to a 64 bit number");
```
(imported from `crate::diag`)